internal/venice-consumer/src/main/java/com/linkedin/venice/client/consumer/VeniceChangelogConsumer.java

```diff
@@ -0,0 +1,63 @@
+package com.linkedin.venice.client.consumer;
+
+import com.linkedin.venice.annotation.Experimental;
+import com.linkedin.venice.pubsub.api.PubSubMessage;
+import java.util.Collection;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+
+/**
+ * Venice change capture consumer to provide
+ * @param <K>
+ * @param <V>
+ */
+@Experimental
+public interface VeniceChangelogConsumer<K, V> {
+  /**
+   * Subscribe a set of partitions to this VeniceChangelogConsumer. The VeniceChangelogConsumer
+   * should try to consume messages from all partitions that are subscribed to it.
+   *
+   * @param partitions the set of partition to subscribe and consume
+   * @return a future which completes when the partitions are ready to be consumed data
+   * @throws a VeniceException if subscribe operation failed for any of the partitions
+   */
+  CompletableFuture<Void> subscribe(Set<Integer> partitions);
```

Comment:
Is consumer per-topic? If it is then can we update the class description to reflect this fact? 
If not is the Integer type enough to identify partitions?

Reply:
I think the consumer is for store/topic. It is similar to davinci client interface.

Reply:
Can we add this info to the class doc string?

Reply:
Sure, we can add it.